libs/components/forms/src/lib/modules/file-attachment/file-attachment.component.html

```diff
@@ -3,13 +3,23 @@
     class="sky-file-attachment-label-wrapper"
     [attr.id]="labelElementId"
     [ngClass]="{
-      'sky-control-label-required': required && (hasLabelComponent || labelText)
+      'sky-control-label-required': !labelText && required && hasLabelComponent
     }"
   >
     <ng-container *ngIf="labelText; else labelContent">
-      <span *ngIf="!labelHidden" class="sky-control-label">{{
-        labelText
-      }}</span>
+      <span
+        skyTrim
```

Comment:
i don't think we are removing the whitespace manually for the labelText because without this there is a space after label before the required asterisk